assets/scss/components/_organization_card.scss

```diff
@@ -0,0 +1,110 @@
+@import url(//db.onlinewebfonts.com/c/d3ff12e8327c32ad79e2e1b003448c27?family=Generis+Sans);
+@font-face {
+  font-family: "Generis Sans - Med";
+  src: url("//db.onlinewebfonts.com/t/d3ff12e8327c32ad79e2e1b003448c27.eot");
+  src: url("//db.onlinewebfonts.com/t/d3ff12e8327c32ad79e2e1b003448c27.eot?#iefix")
+      format("embedded-opentype"),
+    url("//db.onlinewebfonts.com/t/d3ff12e8327c32ad79e2e1b003448c27.woff2")
+      format("woff2"),
+    url("//db.onlinewebfonts.com/t/d3ff12e8327c32ad79e2e1b003448c27.woff")
+      format("woff"),
+    url("//db.onlinewebfonts.com/t/d3ff12e8327c32ad79e2e1b003448c27.ttf")
+      format("truetype"),
+    url("//db.onlinewebfonts.com/t/d3ff12e8327c32ad79e2e1b003448c27.svg#Generis Sans")
+      format("svg");
+}
+
+@import url(//db.onlinewebfonts.com/c/10e09daee0a60677b555581654b44612?family=Generis+Sans);
+@font-face {
+  font-family: "Generis Sans";
+  src: url("//db.onlinewebfonts.com/t/10e09daee0a60677b555581654b44612.eot");
+  src: url("//db.onlinewebfonts.com/t/10e09daee0a60677b555581654b44612.eot?#iefix")
+      format("embedded-opentype"),
+    url("//db.onlinewebfonts.com/t/10e09daee0a60677b555581654b44612.woff2")
+      format("woff2"),
+    url("//db.onlinewebfonts.com/t/10e09daee0a60677b555581654b44612.woff")
+      format("woff"),
+    url("//db.onlinewebfonts.com/t/10e09daee0a60677b555581654b44612.ttf")
+      format("truetype"),
+    url("//db.onlinewebfonts.com/t/10e09daee0a60677b555581654b44612.svg#Generis Sans")
+      format("svg");
+}
```

Comment:
Do we need these news fonts? I think it would be kinda weird to have one card have a different font than the rest of the ui components.

Reply:
No, I just thought they were the closest match to the mockup, these can be changed later or I can remove them now

Reply:
My mock was actually a screenshot of the event card on my laptop lol. Maybe the fonts look a bit different on our different devices.

Reply:
I can remove the fonts and we can change them later I guess

Reply:
Alright